frameworks/helloworld/tests/test_placement.py

```diff
@@ -24,6 +24,48 @@ def configure_package(configure_security):
         sdk_install.uninstall(config.PACKAGE_NAME, config.SERVICE_NAME)
 
 
+@pytest.mark.dcos_min_version("1.12")
+@pytest.mark.sanity
+def test_scheduler_task_placement_by_marathon():
+    sdk_install.uninstall(config.PACKAGE_NAME, config.SERVICE_NAME)
+    # This test ensures that the placement of the scheduler task itself works as expected.
+    some_private_agent = shakedown.get_private_agents().pop()
+    sdk_install.install(
+        config.PACKAGE_NAME,
+        config.SERVICE_NAME,
+        expected_running_tasks=1,
+        additional_options={
+            "service": {
+                "constraints": [["hostname", "CLUSTER", some_private_agent]],
+                "yaml": "simple",
+            }
+        },
+        wait_for_deployment=False,
+    )
+    assert get_task_host(config.SERVICE_NAME) == some_private_agent
+
+
+@pytest.mark.dcos_min_version("1.11")
+@pytest.mark.sanity
+@sdk_utils.dcos_ee_only
+def test_region_zone_injection():
+    def fault_domain_vars_are_present(pod_instance):
```

Comment:
I have nested `fault_domain_vars_are_present` to be inside `test_region_zone_injection` as it is the only place it is used and the logic seemed somewhat contextual to `test_region_zone_injection` only. Comment if this is not idea